src/plugins/deploy/azureDeployPlugin.ts

```diff
@@ -9,7 +9,7 @@ export class AzureDeployPlugin extends AzureBasePlugin {
   public hooks: { [eventName: string]: Promise<any> };
   public commands: any;
 
-  public constructor(serverless: Serverless, options: Serverless.Options) {
+  public constructor(serverless: Serverless, options: Serverless.Options & Serverless.AzureLoginOptions) {
```

Comment:
Will probably need to remove `& Serverless.AzureLoginOptions` here as well to match the base plugin signature.